src/NuGet.Services.BasicSearch/SafeRoleEnvironment.cs

```diff
@@ -62,6 +62,11 @@ public static string GetDeploymentId()
             return string.Empty;
         }
 
+        public static string GetLocalResourceRootPath(string name)
```

Comment:
We need to test what this does when:

1. Size is "small", e.g. 50 GB.
1. Size is large enough to be less than max disk size but too big given already consumed disk space, e.g. 199 GB.
1. Size is exactly what our VM SKU advertises, e.g. 200 GB.
1. Size is great than VM disk size, e.g. 300 GB.

Reply:
Also, what does `cleanOnRoleRecycle` do for us? What will we set it to on our resource?